bim2sim/sim_settings.py

```diff
@@ -1072,48 +1080,43 @@ def __init__(self):
                     "should be plotted.",
         any_string=True
     )
-    # Due to issue #722 this is currently the only way to set AHU values.
-    overwrite_ahu_by_settings = BooleanSetting(
-        default=True,
-        description='Overwrite central AHU settings with the following '
-                    'settings.',
-    )
-    ahu_heating = BooleanSetting(
-        default=False,
+    ahu_heating_overwrite = BooleanSetting(
+        default=None,
         description="Choose if the central AHU should provide heating. "
-                    "Set overwrite_ahu_by_settings to True, "
-                    "otherwise this has no effect. "
     )
-    ahu_cooling = BooleanSetting(
-        default=False,
+    ahu_cooling_overwrite = BooleanSetting(
+        default=None,
         description="Choose if the central AHU should provide cooling."
-                    "Set overwrite_ahu_by_settings to True, "
-                    "otherwise this has no effect. "
     )
-    ahu_dehumidification = BooleanSetting(
-        default=False,
+    ahu_dehumidification_overwrite = BooleanSetting(
+        default=None,
         description="Choose if the central AHU should provide "
                     "dehumidification."
-                    "Set overwrite_ahu_by_settings to True, "
-                    "otherwise this has no effect. "
     )
-    ahu_humidification = BooleanSetting(
-        default=False,
+    ahu_humidification_overwrite = BooleanSetting(
+        default=None,
         description="Choose if the central AHU should provide humidification."
-                    "Set overwrite_ahu_by_settings to True, "
                     "otherwise this has no effect. "
     )
-    ahu_heat_recovery = BooleanSetting(
-        default=False,
+    ahu_heat_recovery_overwrite = BooleanSetting(
+        default=None,
         description="Choose if the central AHU should zuse heat recovery."
-                    "Set overwrite_ahu_by_settings to True, "
-                    "otherwise this has no effect. "
     )
-    ahu_heat_recovery_efficiency = NumberSetting(
-        default=0.65,
+    ahu_heat_recovery_efficiency_overwrite = NumberSetting(
+        default=None,
         min_value=0.5,
         max_value=0.99,
         description="Choose the heat recovery efficiency of the central AHU."
-                    "Set overwrite_ahu_by_settings to True, "
-                    "otherwise this has no effect. "
+    )
+    use_constant_infiltration_overwrite = BooleanSetting(
```

Comment:
Is there an additional Parameter in the use conditions defining the constant infiltration? If yes, please add it.

Reply:
no